### ISSUE TITLE
Take <h1>, <h2>, ... tags out of typography components and define them on the page

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1398]
use:

`<h1><header1 /></h1>`

on the page, where header is a styled `<span>`, rather than:

`<header1 />`

where header contains an `<h1>`.

Relevant discussion from Slack:

https://webdesign.tutsplus.com/articles/the-importance-of-heading-levels-for-assistive-technology--cms-31753

"The real world may call for a fifth level heading that needs to look like it is a second-level heading, or something entirely new. In these situations it’s better to preserve the underlying logical page structure and modify the heading’s visuals instead".

That tells me it's less important for an `<h2>` to look the same everywhere. And looking on gov.uk, I just noticed they have `<h2>`s on the same page that are styled very differently, from 16px font to 36px.

So we should just specify the heading level case by case, and not build components that are the same heading level everywhere.

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility work](https://trello.com/c/H7Q1Ycrm/93-accessibility-work)

**Comment by smcmurtry** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Accessibility: Screenreader Navigation](https://trello.com/c/knKtuM4T/100-accessibility-screenreader-navigation)
